lib/Twig/Environment.php

```diff
@@ -494,6 +494,18 @@ public function isTemplateFresh($name, $time)
     }
 
     /**
+     * Returns up-to-date timestamp of all template loaded.
+     *
+     * @return string
+     */
+    public function getTemplateLoaderFreshness()
```

Comment:
Btw, even the name is weird. This does not return a "freshness" (what is this?) but a timestamp.